packages/@aws-cdk/aws-ecs/test/ec2/test.ec2-service.ts

```diff
@@ -25,7 +26,7 @@ export = {
 
       new ecs.Ec2Service(stack, "Ec2Service", {
         cluster,
-        taskDefinition
+        taskDefinition,
```

Comment:
nit: Can you remove the extra commas after taskDefinition this appears in multiple places?

Reply:
yea of course. I was not sure what's the convention because I've seen both (i.e. with trailing comma and not)

Reply:
I'd say if there are no arguments after the fact, no trailing commas. @rix0rrr @eladb  thoughts?

Reply:
In TypeScript trailing commas are very common. The big benefit of them is that if you add lines (properties, objects, etc) you do not forget them and in git the preceding line does not change which makes diffs cleaner.

Reply:
Oops did not see the last comment. Hmm, if trailing commas are common then I would just revert my last commit.